### ISSUE TITLE
Also fall back to future versions of fusermount

### ISSUE THREAD

**Issue by probonopd** [azubieta/appimage-runtime#13]
https://github.com/AppImageCrafters/appimage-runtime/blob/main/patches/fusermount2_fallback.patch falls back to `fusermount` in case `fusermount3` is not found.

This helps for systems that have libfuse2 but not libfuse3 installed.

But what about future systems that will have libfuse4, 5, ... n instead of libfuse3?

Maybe we should fall back to those as well:

* Try fusermount3
* If not available, fall back to fusermount
* If not available, fall back to fusermount4
* If not available, fall back to fusermount5
* ...
* If not available, fall back to fusermountN

Yes, it's "based on hope", but it'd be no worse than not even trying...

**Comment by azubieta** [comment]
"Hopefully" distributions will keep having a `fusermount` binary for backward compatibility. Regarding new version we will hate to wait a see how they name the binary.

Maybe for fuse4 there will be no need of having a `fusermount` binary

**Comment by probonopd** [comment]
Yes, but it wouldn't hurt to search for future ones, would it? Worst thing that can happen is that it won't work, like if we don't search for them.
